sat/cli/jobstat/main.py

```diff
@@ -44,18 +46,23 @@ def wrap_long_row_lines(row):
         None. Modifies row in place.
     """
     if row.get('apid'):
-        row['apid'] = textwrap.fill(text=row['apid'], width=20)
+        row['apid'] = row['apid'][:8]
```

Comment:
Just checking, was it intentional to remove the trailing "..." now? My suggestion was just to change the spacing around the `+` operator, but I'm fine with this as well.

Reply:
this was recommended by our team; as first 8 chars is good enough.